modules/lambda-adapter/package.json

```diff
@@ -0,0 +1,34 @@
+{
+  "name": "@phenyl/lambda-adapter",
+  "version": "1.0.0",
+  "description": "Adapter for AWS lambda",
+  "repository": "phenyl-js/phenyl",
+  "license": "Apache License 2.0",
+  "author": "Shin Suzuki <shinout310@gmail.com>, Jesse Katsumata <jesse.katsumata@gmail.com>",
+  "files": [
+    "lib",
+    "LICENSE",
+    "README.md"
+  ],
+  "main": "./lib/index.js",
+  "types": "lib/index.d.ts",
+  "scripts": {
+    "build": "upbin tsc --declaration",
+    "ci-test": "upbin nyc npm test --color always",
+    "clean": "upbin rimraf $(cat ../../.temporary-files)",
+    "lint": "upbin eslint 'src/**/*.ts' --fix",
+    "test": "upbin mocha --require ts-node/register 'test/**/*.test.ts' --color always",
+    "type-check": "upbin tsc --noEmit",
+    "watch": "upbin tsc --declaration --watch"
+  },
+  "dependencies": {
+    "@phenyl/interfaces": "~1.0.0",
```

Comment:
I think we should take advantage of Lerna,
how about this?
```
@phenyl/interfaces: "*",
@phenyl/utils: "*",
@phenyl/http-rules: "*",
```

Reply:
version number of lerna monorepo packages are automatically updated using lerna publish.
This command also updates the version number on package.json, so we can continue using the exact version number.
https://github.com/lerna/lerna/tree/master/commands/publish